### ISSUE TITLE
[R, Windows] Two demos of mxnet have Errors

### ISSUE THREAD

**Issue by fengfengyuyu** [dmlc/mxnet#631]
I just installed the windows version of mxnet. Platform: x86_64-w64-mingw32/x64 (64-bit)
I tried demos. Two demos are not available as follows:

> demo(basic_executor,package="mxnet")

> require(mxnet)
> A = mx.symbol.Variable('A')
> B = mx.symbol.Variable('B')
> C = A + B
> a = mx.nd.zeros(c(2), mx.cpu())
> b = mx.nd.array(as.array(c(1, 2)), mx.cpu())
> exec = mx.symbol.bind(
  symbol=C,
  ctx=mx.cpu(),
  arg.arrays = list(A=a, B=b),
  aux.arrays = list(),
  grad.reqs = list(FALSE, FALSE))

**Error in eval(expr, envir, enclos) : 
  could not find function "mx.symbol.bind"**

![basic_excutor](https://cloud.githubusercontent.com/assets/11608887/11263838/85a8ee12-8ec9-11e5-976a-05d1356bcbde.jpg)

Another demo is:
> demo(basic_model,package="mxnet")
> require(mxnet)
> batch.size <- 100
> data <- mx.symbol.Variable("data")
> fc1 <- mx.symbol.FullyConnected(data, name="fc1", num_hidden=128)

> act1 <- mx.symbol.Activation(fc1, name="relu1", act_type="relu")

> fc2 <- mx.symbol.FullyConnected(act1, name = "fc2", num_hidden = 64)

> act2 <- mx.symbol.Activation(fc2, name="relu2", act_type="relu")

> fc3 <- mx.symbol.FullyConnected(act2, name="fc3", num_hidden=10)

> softmax <- mx.symbol.Softmax(fc3, name = "sm")

> dtrain = mx.io.MNISTIter(
   image="data/train-images-idx3-ubyte",
   label="data/train-labels-idx1-ubyte",
   data.shape=c(784),
   batch.size=batch.size,
   shuffle=TRUE,
   flat=TRUE,
   silent=0,
   seed=10)

[13:43:14] d:\chhong\mxnet\src\operator\./softmax_output-inl.h:187: Softmax symbol is renamed to SoftmaxOutput. This API will be deprecated in Dec, 2015
[13:43:14] D:\chhong\mxnet\dmlc-core\include\dmlc/logging.h:208: [13:43:14] D:\chhong\mxnet\dmlc-core\src\io\local_filesys.cc:149: Check failed: allow_null  LocalFileSystem: fail to open "data/train-images-idx3-ubyte"
**Error in eval(expr, envir, enclos) : 
  [13:43:14] D:\chhong\mxnet\dmlc-core\src\io\local_filesys.cc:149: Check failed: allow_null  LocalFileSystem: fail to open "data/train-images-idx3-ubyte"**

![basic_model](https://cloud.githubusercontent.com/assets/11608887/11263834/7b14f842-8ec9-11e5-9785-eaf4571f0118.jpg)

**Comment by thirdwing** [comment]
The binary package is from a Japanese friend: https://github.com/datasciesotist/files
